frontend/src/Components/Elements/Table/TableModal.js

```diff
@@ -322,6 +330,49 @@ function TableModal({ chartState, dataSet, columns, isOpen, closeModal }) {
     return data;
   };
 
+  const missingPhraseForGraph = (ds) => {
+    const phrase = {
+      STOPS: 'stops',
+      SEARCHES: 'searches',
+      STOPS_BY_REASON: 'stops',
+      SEARCHES_BY_TYPE: 'searches',
+      USE_OF_FORCE: 'use of force',
+      CONTRABAND_HIT_RATE: 'contraband',
+      LIKELIHOOD_OF_SEARCH: 'searches',
+    }[ds];
+    return `No ${phrase} have been reported`;
+  };
+
+  const setupCSVData = (ds) => {
+    const csvData = JSON.parse(JSON.stringify(_buildTableData(ds)));
+    // Cleanup data tables for spreadsheet download
+    csvData.forEach((datum) => {
+      if (datum.hasOwnProperty('no_data') && datum.no_data) {
```

Comment:
This function makes me want to refactor this and all incoming data manipulation to the backend and cleanup the logic. Maybe at a later time.